pixo-platform/blobstorage/gcs/urls.go

```diff
@@ -64,6 +64,15 @@ func (c Client) GetSignedURL(ctx context.Context, object blobstorage.UploadableO
 		GoogleAccessID: conf.Email,
 		PrivateKey:     conf.PrivateKey,
 	}
+	if len(options) > 0 {
+		option := options[0]
+
+		if option.ContentDisposition != "" {
```

Comment:
thoughts on adding expire duration at this point ? should be easy enough and probably something the client should have control over

Reply:
Resolved.